password/impl/src/main/java/org/wildfly/security/password/impl/PasswordFactorySpiImpl.java

```diff
@@ -498,8 +496,7 @@ protected Password engineGeneratePassword(final String algorithm, final KeySpec
                 break;
             }
         }
-        throw log.invalidKeySpecUnknownAlgorithmOrIncompatiblePasswordSpec();
-    }
+        throw log.invalidKeySpecUnknownAlgorithmOrIncompatiblePasswordSpec(algorithm, keySpec == null ? null : keySpec.getClass().getSimpleName());    }
```

Comment:
Just a minor, but closing } should  be on new line

Reply:
Fixed